learning_journal/tests/test_models.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+from learning_journal.models import Entry, DBSession
+
+def test_create_entry(dbtransaction):
+    """Assert entry was entered into database."""
+    new_entry = Entry(title="Entry1", text="Hey, this works. Awesome.")
+    assert new_entry.id is None
+    DBSession.add(new_entry)
+    DBSession.flush()
+    assert new_entry.id is not None
```

Comment:
This is a good start.  But where could you go from here?  What other constraints on your Entry are there?  Can you derive ways to test that the database appropriately throws errors if you try to create entries that violate those constraints?